src/main/java/edu/wpi/grip/core/operations/composite/HSVThresholdOperation.java

```diff
@@ -21,7 +21,7 @@
 
     private final SocketHint<opencv_core.Mat> inputHint = new SocketHint<opencv_core.Mat>("Input", opencv_core.Mat.class, opencv_core.Mat::new);
     private final SocketHint<List> hueHint = new SocketHint<List>("Hue", List.class,
-            () -> Arrays.asList(0.0, 255.0), SocketHint.View.RANGE, new List[]{Arrays.asList(0.0, 180.0)});
+            () -> Arrays.asList(0.0, 180.0), SocketHint.View.RANGE, new List[]{Arrays.asList(0.0, 180.0)});
```

Comment:
Your comment made me remember that I didn't fix this. This is in a separate commit. Hope you don't mind